roles/openshift_register_nodes/tasks/main.yml

```diff
@@ -19,7 +19,7 @@
 - name: Create node config
   command: >
     /usr/bin/openshift admin create-node-config
-      --node-dir={{ openshift_cert_dir }}/node-{{ item.openshift.common.hostname }}
+      --node-dir={{ openshift_master_config_dir }}/node-{{ item.openshift.common.hostname }}
```

Comment:
Do we really want to put the node config directories into openshift_master_config_dir?

Maybe an additional directory under /etc/openshift specifically for node configs?  possibly /etc/openshift/generated-node-configs ? or maybe even /etc/openshift/generated-configs/